src/main/java/budgetbuddy/MainApp.java

```diff
@@ -76,14 +77,20 @@ public void init() throws Exception {
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         LoansManager loansManager;
+        AccountsManager accountsManager;
         Optional<ReadOnlyAddressBook> addressBookOptional;
         ReadOnlyAddressBook initialData;
-        loansManager = new LoansManager(); // TODO Load loan data from storage (if data present).
+        loansManager = new LoansManager();
+        accountsManager = new AccountsManager();
+        // TODO Load loan data from storage (if data present).
         try {
             addressBookOptional = storage.readAddressBook();
             if (!addressBookOptional.isPresent()) {
                 logger.info("Data file not found. Will be starting with a sample Budget Buddy");
             }
+            /**if (!accountBookOptional.isPresent()) {
+                logger.info("Data file not found. Will be starting with a sample Budget Buddy");
+            }*/
```

Comment:
Duplicate of the previous 3 lines?

Reply:
Excellent observation. They are already deleted :)